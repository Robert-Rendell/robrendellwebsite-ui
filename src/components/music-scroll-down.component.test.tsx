import React from "react";
import { render } from "@testing-library/react";
import { MusicScrollDownMessage } from "./music-scroll-down.component";

describe("MusicScrollDownMessage component", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<MusicScrollDownMessage/>);
    expect(asFragment()).toMatchSnapshot();
  });
});