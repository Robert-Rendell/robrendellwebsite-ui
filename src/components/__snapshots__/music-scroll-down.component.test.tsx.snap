// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MusicScrollDownMessage component should match snapshot 1`] = `
<DocumentFragment>
  <p>
    <i>
      Scroll down to the 
      <a
        class="scroll-to-bottom"
      >
        bottom
      </a>
       of this page and play the youtube video if you want music while reading :)
    </i>
  </p>
  <div
    style="cursor: pointer;"
  >
    <span
      style="color: orange;"
    >
      Disable youtube autoplay?
    </span>
    <svg
      fill="currentColor"
      height="40"
      style="color: white; padding: 7px;"
      viewBox="0 0 16 16"
      width="40"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M8.5 2a.5.5 0 0 1 .5.5v11a.5.5 0 0 1-1 0v-11a.5.5 0 0 1 .5-.5zm-2 2a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7a.5.5 0 0 1 .5-.5zm4 0a.5.5 0 0 1 .5.5v7a.5.5 0 0 1-1 0v-7a.5.5 0 0 1 .5-.5zm-6 1.5A.5.5 0 0 1 5 6v4a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm8 0a.5.5 0 0 1 .5.5v4a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm-10 1A.5.5 0 0 1 3 7v2a.5.5 0 0 1-1 0V7a.5.5 0 0 1 .5-.5zm12 0a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0V7a.5.5 0 0 1 .5-.5z"
        fill-rule="evenodd"
      />
    </svg>
  </div>
  <p />
  <hr />
  <span>
    Back toÂ 
    <a
      class="scroll-to-top"
    >
      top of page
    </a>
    .
  </span>
</DocumentFragment>
`;
